electron/generators/diograph-generator.spec.js

```diff
@@ -0,0 +1,164 @@
+const { readPaths } = require('../readers/folder-reader')
```

Comment:
Well, I would have liked to test many modules at once but it caused problems and I wanted to spec the behavior better so that's why I choosed to mock most of submodules.

Reply:
So, I wouldn's say this is the only correct way to do tests/specs.